src/components/Filter.tsx

```diff
@@ -64,6 +64,7 @@ export function Filter({ setSearch, setTimezone, state, toggleTag }: Filter) {
           {open ? 'Close' : 'Filters'}
         </Button>
       </FormControl>
+      <Textual></Textual>
```

Comment:
I'm not happy with this, but it works. There is probably a better way to add a text component.